docs-ref-services/servicebus.md

```diff
@@ -51,20 +204,20 @@ sbs = ServiceBusService(service_namespace,
                         shared_access_key_value=key_value)
 ```
 
-### ACS Authentication
+#### ACS Authentication
```

Comment:
What's "ACS" authentication? At first mention (or second mention in this case if you'd like to keep the heading as-is), always use the full term, and you can add "(ACS)" at the end and refer to ACS from then on.